readme.md

```diff
@@ -1,14 +1,138 @@
+Run `npm install` or `yarn` to install the necessary dependencies.
+
+### Step 1 - Create the SSCCE
+
+Modify [src/sscce.ts](./src/sscce.ts) to create a failing test that accurately represents the issue you're experiencing.
+
+Remember the [SSCCE rules](http://www.sscce.org/). It should be:
+
+- *Short (Small)* - It should not include anything that is not relevant to your issue.
+- *Self Contained* - Ensure everything is included, ready to go.
+- *Correct* - It should demonstrate the problem you're encountering (i.e. the sscce should fail, but with the right error).
+- *Example* - Displays the problem we are trying to solve.
+
+### Step 2 - Run the SSCCE
+
+Use one of our npm scripts to run your SSCCE.
+
+Running with a dialect other than sqlite will require installing an extra package
+& having a database running. For convenience, we provide docker containers for each database.
+
+You can also opt to open a pull request, and run the code directly in our CI.
+
+#### sqlite
+
+The easiest of the bunch. Simply run:
+
+```shell
+npm run test:sqlite
+```
+
+#### postgres
+
+You'll need to install the `pg` package and have a postgres database running.
+
+```shell
+# Do this only once.
+npm install pg
+# or
+npm install pg-native
+
+# Start the postgres database using docker.
+# Requires docker. You can also run your own database if you prefer.
+npm run start:postgres
+
+# run the sscce!
+npm run test:postgres
+# or
+npm run test:postgres-native
+
+# Remember to stop the docker container once you're done.
+npm run stop:postgres
+```
+
+#### mariadb
+
+```shell
+# Do this only once.
+npm install mariadb
+
+# Start the mariadb database using docker.
+# Requires docker. You can also run your own database if you prefer.
+npm run start:mariadb
+
+# run the sscce!
+npm run test:mariadb
+
+# Remember to stop the docker container once you're done.
+npm run stop:mariadb
+```
+
+#### mysql
+
+```shell
+# Do this only once.
+npm install mysql2
+
+# Start the mysql database using docker.
+# Requires docker. You can also run your own database if you prefer.
+npm run start:mysql
+
+# run the sscce!
+npm run test:mysql
+
+# Remember to stop the docker container once you're done.
+npm run stop:mysql
+```
+
+#### mssql (SQL Server)
+
+```shell
+# Do this only once.
+npm install tedious
+
+# Start the mssql database using docker.
+# Requires docker. You can also run your own database if you prefer.
+npm run start:mssql
+
+# run the sscce!
+npm run test:mssql
+
+# Remember to stop the docker container once you're done.
+npm run stop:mssql
+```
+
+### Step 3 - Commit your SSCCE & sent it to us
+
+Open an issue on the [main sequelize repo](https://github.com/sequelize/sequelize/) describing
+your problem and include a link to your SSCCE in it.
+
+You can also send a fork to the [sequelize-sscce](https://github.com/sequelize/sequelize-sscce) repository,
```

Comment:
send a fork to this repo? What do you mean with that?

Reply:
this was meant to be "of the repository", and would be better phrased as "You can also include a fork of the repository"